wallet/wallet.go

```diff
@@ -3469,27 +3496,8 @@ func (w *Wallet) reliablyPublishTransaction(tx *wire.MsgTx,
 	// We'll also ask to be notified of the transaction once it confirms
 	// on-chain. This is done outside of the database transaction to prevent
 	// backend interaction within it.
-	//
-	// NOTE: In some cases, it's possible that the transaction to be
-	// broadcast is not directly relevant to the user's wallet, e.g.,
-	// multisig. In either case, we'll still ask to be notified of when it
-	// confirms to maintain consistency.
-	//
-	// TODO(wilmer): import script as external if the address does not
-	// belong to the wallet to handle confs during restarts?
-	for _, txOut := range tx.TxOut {
-		_, addrs, _, err := txscript.ExtractPkScriptAddrs(
-			txOut.PkScript, w.chainParams,
-		)
-		if err != nil {
-			// Non-standard outputs can safely be skipped because
-			// they're not supported by the wallet.
-			continue
-		}
-
-		if err := chainClient.NotifyReceived(addrs); err != nil {
-			return nil, err
-		}
+	if err := chainClient.NotifyReceived(ourAddrs); err != nil {
```

Comment:
Will this still be okay in the case where we send a 1 output tx to a destination that is not ours? We don't rely on confirmation being anywhere for that case?

Reply:
Good question, it should be notified as any addresses we're watching that get spent also request a notification on the input spending it for when it confirms.